src/helpers.ts

```diff
@@ -64,7 +53,11 @@ export const isCoordinateInPolygon = (coordinate: Coordinate, polygon: Coordinat
     return inside;
```

Comment:
Suggestion: what about returning the intersect value, so basically `return intersect`! is that possible rather than introducing inside variable.

Reply:
No, please take a look on the source of this algorithm https://wrf.ecse.rpi.edu//Research/Short_Notes/pnpoly.html 😄